spec/rspec/matchers/be_record_matcher_spec.rb

```diff
@@ -1,6 +1,29 @@
-require "spec_helper"
-require "support/controller_specs_boilerplate"
+require 'spec_helper'
+require 'support/controller_specs_boilerplate'
+require 'rspec/matchers/fail_matchers'
 
-describe "be_record matcher", type: :controller do
+class User < ActiveRecord::Base; end
+class ChildUser < User; end
 
+describe 'be_record matcher', type: :controller do
```

Comment:
It was here before and I thought that it was good.  https://github.com/Selleo/rspec-controller-matchers/blob/master/spec/rspec/matchers/be_record_matcher_spec.rb